Helpers/EmailSender.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mail;
+using System.Threading.Tasks;
+
+namespace FoodKept.Helpers
+{
+    public class EmailSender
+    {
+        private SmtpClient smtpClient;
+
+        public EmailSender()
+        {
+            smtpClient = createClient();
+        }
+
+        public void sendEmail(string email, string body)
+        {
+            MailMessage mail = new MailMessage();
+            mail.IsBodyHtml = true;
+            mail.From = new MailAddress("foodkepterino@gmail.com", "FoodKept");
+            mail.To.Add(new MailAddress("foodkepterino@gmail.com"));
```

Comment:
Could have commented that this will need to be changed to customer or restaurant email.

Reply:
This email is sent to our page's admin (with email "foodkepterino@gmail.com), so he could confirm that, for example, that restaurant exists. So everything is good there

Reply:
Ok understood